exporter/exporterhelper/common.go

```diff
@@ -86,6 +86,19 @@ func WithRetry(config configretry.BackOffConfig) Option {
 	}
 }
 
+// WithConcurrency overrides the default ConcurrencySettings for the exporter.
+// The default ConcurrencySettings is to enable concurrency with a limit of 10 goroutines.
+func WithConcurrency(config ConcurrencySettings) Option {
+	return func(o *baseExporter) error {
+		if !config.Enabled {
+			o.exportFailureMessage += " Try enabling the concurrency config option to send more requests in parallel."
```

Comment:
This seems misleading. Not sure how enabling concurrency can help with export errors. There is no concurrency limit by default

Reply:
Simplified the config so this error is now removed